src/AutoNumeric.js

```diff
@@ -7337,17 +7390,30 @@ To solve that, you'd need to either set \`decimalPlacesRawValue\` to \`null\`, o
                     case AutoNumeric.options.emptyInputBehavior.focus:
                         setValue = false;
                         break;
-                    //TODO What about the `AutoNumeric.options.emptyInputBehavior.press` value?
+                    case AutoNumeric.options.emptyInputBehavior.null:
```

Comment:
After checking your changes here, it occurred to me that I was setting `setValue` for no apparent reason in that block!

You can scrap those lines are just use the following instead:
```js
if (currentValue === '') {
    switch (this.settings.emptyInputBehavior) {
        case AutoNumeric.options.emptyInputBehavior.focus:
        case AutoNumeric.options.emptyInputBehavior.null:
        case AutoNumeric.options.emptyInputBehavior.press:
            break;
        case AutoNumeric.options.emptyInputBehavior.always:
            this._setElementValue(this.settings.currencySymbol);
            break;
        case AutoNumeric.options.emptyInputBehavior.min:
            this.set(this.settings.minimumValue);
            break;
        case AutoNumeric.options.emptyInputBehavior.max:
            this.set(this.settings.maximumValue);
            break;
        case AutoNumeric.options.emptyInputBehavior.zero:
            this.set('0');
            break;
        // When emptyInputBehavior is a number or a string representing a number
        default:
            this.set(this.settings.emptyInputBehavior);
    }
} else if (setValue && currentValue === this.domElement.getAttribute('value')) {
    this.set(currentValue);
}
```